docs/pages/2019_MARVEL_Psik_MaX/sections/verdi_cmdline.rst

```diff
@@ -39,406 +54,385 @@ The first output line should look like
+Once a process is in a terminal state, it will never become active again.
+The `official documentation <https://aiida-core.readthedocs.io/en/latest/concepts/processes.html#process-state>`_ contains more details on process states.
+
+If, instead of any one particular process state, you want to list processes in *any* state, you can use the ``-a/--all`` flag instead:
+
+.. code:: console
+
+    verdi process list -a
+
+This command will list all the processes that have *ever* been launched.
+As your database will grow, the output will become so large that it becomes unusable.
+To limit the number of results, you can use the ``-p/--past-days <NUM>`` option, that will only show processes that were created ``NUM`` days ago.
+For example, if you want to list all the process launched since the last day, you can execute:
 
 .. code:: console
 
-    verdi process list --process-state finished
+    verdi process list -a -p1
 
-Another very typical option combination allows to get processes in
-*any* state (flag ``-a/--all``) generated in the past ``NUM`` days
-(``-p/--past-days <NUM>``): e.g., for process in the past 1 day:
-``verdi process list -p1 -a``. Each row of the output identifies a process and
-shows some information about it. For a more detailed list of properties, choose
-one row by noting down its PK (primary key) number (first column of the output)
-and type in the terminal
+Each row of the output identifies a process with some basic information about its status.
+For a more detailed list of properties, you can use ``verdi process show``, but to address any specific process, you need an identifier for it.
+An identifier for any entity in AiiDA comes in three different forms:
+
+ * PK
+ * UUID
+ * LABEL
+
+The first one, the PK or primary key, you have already seen.
+These are the integers in the first column of the output printed by ``verdi process list``.
+The PK is a unique identifier for any entity, but only for that *entity type* and *within your database*.
+That is why, in addition to their PK, all AiiDA entities also have a UUID (Universal Unique Identifier), which as the name suggests, should be unique, `even across databases`, and will never change.
+That is to say, if you share some of your data with somebody else, the UUIDs will remain identical, unlike the PKs which will most likely change.
+
+.. note::
+
+    Any ``verdi`` parameter that expects an identifier will accept either a PK, a UUID or a label, although, not all entities will have a label, as it is an optional field.
+    For a UUID you can even specify only a part of it, as long as it starts at the beginning and the partial can be uniquely resolved.
+    For more information on identifiers in ``verdi`` and AiiDA in general, refer to the `documentation online <https://aiida-core.readthedocs.io/en/latest/verdi/verdi_user_guide.html#cli-identifiers>`_.
+
```

Comment:
This section very naturally raises the question: why PKs? We should add a note on this.
Something like

```
.. note ::
      You may be wondering why AiiDA doesn't rely just on UUIDs.
      Besides the historical fact that PKs came first, UUIDs need to encode more information in order to be universally unique. While we recommend using UUIDs wherever they feel comfortable, PKs are shorter and can be easier to work with.

Reply:
Done: added it to the note that was already discussing some more in depth reasoning